File: retroauth-android/src/test/java/com/andretietz/retroauth/AndroidTokenTest.java
package com.andretietz.retroauth;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.junit.runners.JUnit4;

import static org.junit.Assert.assertEquals;


@RunWith(JUnit4.class)
public class AndroidTokenTest {

    @Test
    public void dataCheck() {
        AndroidToken androidToken = new AndroidToken("token", "refresh");
        assertEquals("token", androidToken.token);
        assertEquals("refresh", androidToken.refreshToken);
    }
}

File: retroauth-android/src/test/java/com/andretietz/retroauth/AndroidTokenTest.kt
package com.andretietz.retroauth

import org.junit.Assert.assertEquals
import org.junit.Test
import org.junit.runner.RunWith
import org.junit.runners.JUnit4

@RunWith(JUnit4::class)
class AndroidTokenTest {

  @Test
  fun dataCheck() {
    val androidToken = AndroidToken("token", mapOf("refresh" to "refresh"))
    assertEquals("token", androidToken.token)
    assertEquals("refresh", androidToken.data!!["refresh"])
  }
}
